src/admin/views/UsersView.js

```diff
@@ -143,6 +155,12 @@ const UsersView = () => {
         <Form widths="equal">
           <Form.Group inline>
             <span className="pr-5">Filter by:</span>
+            <Checkbox
```

Comment:
I wonder if we should display something that explains what this filter does on hover? Like something that says  when checked this shows users that are not in the selected org OR in the selected group. Bc its not clear if this operation is a NOT org or NOT group or if its doing NOT ( org and group )